ui/src/flux/helpers/autoComplete.ts

```diff
@@ -124,7 +124,7 @@ const shouldCompleteFunction = (currentLineText, cursorPosition) => {
     const char = currentLineText[i]
     const charBefore = currentLineText[i - 1]
 
-    if (charBefore + char === pipe || char === endOfFunc) {
+    if (charBefore + char === pipe || charBefore + char === endOfFunc + ' ') {
```

Comment:
what do you think about doing `(charBefore + char).trim() === pipe`?

Reply:
I think that would match the case when a user has typed `Space` `)` as well, which we want to avoid. It would show the autocomplete immediately after a user finishes typing a function, which prevents them from pressing `Enter` to write the next function on a new line.